WhaleSpotting/react-app/src/styles/Profile.scss

```diff
@@ -0,0 +1,133 @@
+@import './Constants.scss';
+
+.body {
+    max-width: 1024px;
+    font-family: $fontFamily;
+    margin: 0 auto 0 auto;
+    display: flex;
+    flex-direction: row;
+}
+
+.profile-pane {
+    height: 100vh;
+    width: 250px;
+    background-color: $secondaryDark;
+}
+
+.feed{
+    padding: 0px 20px;
+    flex: 1;
+}
+
+.outer-container {
+    width: 90%;
+    flex-direction: column;
+}
+
+.little-text{
+    font-size: $bodyFontSize;
+    margin: 0px;
+}
+
+.joined{
+    margin-top:-7px;
+}
+
+.reported{
+    margin-left: 10px;
+}
+
+.feature-text{
+    font-size: $featureFontSize;
+    margin:0px;
+}
+
+.heading{
+    font-size: $titleFontSize;
+    margin:0px;
+}
+
+.trophy {
+    width: 80px;
+}
+
+.profile-image {
+    width: 100%;
+}
+
+.outer-container{
+    display:flex;
+    flex-direction: column;
+    width: 90%;
+    margin-left:auto;
+    margin-right:auto;
+}
+
+.button-container{
+    display:flex;
+    flex-direction: column-reverse;
+    flex: 1;
+    align-items: center;
+}
+
+.trophy-container{
+    display:flex;
+    flex-direction: row;
+    align-items:center;
+}
+
+.trophy-image{
+    width:70px;
+    height:auto;
```

Comment:
This does nothing AFAICT

Reply:
changed to 70px to ensure always a circle. This allows us flexibility to create badges at different pixel sizes as 70px is a bit small.